### PR TITLE
Feat: amend activeness logic for activeness

### LINKED ISSUE #284: Problem: Crossfire activeness task logic need amendment
Primary Node must be setup on phase 1. And any activeness task should check task_phase_1_node_setup. Including:

- Entry level  always up task (during phase2)
- MVP up time task (during phase2)
- MVP up time task (during phase3)

### PR DESCRIPTION
**Pull request by foreseaz**
Disable taskActive and commitment ranking for the validator who missed taskSetup, resolved #284

### CODE REVIEW

File: appinterface/projection/crossfire/crossfire.go [hunk 1]
```diff
@@ -472,6 +472,12 @@ func (projection *Crossfire) checkTaskKeepActive(
 	}
 
 	for _, validator := range validators {
+		// If validator missed the task 1, it cannot complete the active task
+		if validator.TaskPhase1NodeSetup == constants.MISSED {
+			projection.logger.Debugf("Won't handle task keepActive for %s because it missed taskSetup", validator.OperatorAddress)
```
Comment (cdc-Hitesh) [inline appinterface/projection/crossfire/crossfire.go:477]: This Log goes in the chain-indexing logs. 
Is that behaviour fine?

File: appinterface/projection/crossfire/crossfire.go [hunk 2]
```diff
@@ -581,9 +587,28 @@ func (projection *Crossfire) computeCommitmentRank(
...
+		// check whether validator missed taskSetup or not
+		validatorRow, err := crossfireValidatorView.FindBy(view.CrossfireValidatorIdentity{
+			MaybeOperatorAddress: &statsOperatorAddress,
+		})
+		if errors.Is(err, rdb.ErrNoRows) {
+			// no validator found by tendermint address
+			continue
+		}
+		if err != nil {
+			return fmt.Errorf("error getting validator with operator addr %s %v", statsOperatorAddress, err)
```
Comment (cdc-Hitesh) [inline appinterface/projection/crossfire/crossfire.go:604]: This will revert the whole process and will stop. Is that intended?

**Review by cdc-Hitesh — COMMENTED** [review]
Left some comments. Otherwise looks good

File: appinterface/projection/crossfire/crossfire.go (hunk 2 above)
Comment (foreseaz) [reply appinterface/projection/crossfire/crossfire.go:604]: Yes, should be~ if the error is non-norow error, then it's a big problem and should be fixed by us

File: appinterface/projection/crossfire/crossfire.go (hunk 1 above)
Comment (foreseaz) [reply appinterface/projection/crossfire/crossfire.go:477]: I think it should be fine

**Review by cdc-Hitesh — APPROVED** [review]

**Pull request merged**